lib/ansible/plugins/connection/docker.py

```diff
@@ -132,8 +132,8 @@ def _get_docker_version(self):
 
         cmd, cmd_output, err, returncode = self._old_docker_version()
         if returncode == 0:
-            for line in cmd_output.split('\n'):
-                if line.startswith('Server version:'):  # old docker versions
+            for line in cmd_output.split(b'\n'):
+                if line.startswith(b'Server version:'):  # old docker versions
```

Comment:
Wouldn't it be better to pass cmd_output through decode?  Therefore, we don't need to handle bytes directly?

```
cmd_output = cmd_output.decode('utf-8')
```

Then we can keep the code exactly as is.